crates/standalone/src/lib.rs

```diff
@@ -46,17 +46,11 @@ pub struct StandaloneEnv {
     client_actor_index: ClientActorIndex,
     public_key: DecodingKey,
     private_key: EncodingKey,
-
-    /// Whether databases in this environment will be created entirely in memory
-    /// or otherwise persist their message log and object store to disk.
-    ///
-    /// Note that this does not apply to the StandaloneEnv's own control_db
-    /// or object_db.
-    storage: Storage,
+    config: Config,
```

Comment:
I do think a bit of the previous comment is worth persisting here.

In particular, right now we configure these things *at the level of the whole environment*, not *at the level of the database*. I can easily imagine wanting the latter eventually.